deploy/skaffold/builder.dockerfile

```diff
@@ -12,120 +12,106 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FROM gcr.io/gcp-runtimes/ubuntu_16_0_4
+
```

Comment:
nit: `ENV DOCKER_VERSION 18.09.6`

and then

`COPY --from=docker:${DOCKER_VERSION}`